CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.1.0
+### Fixed
```

Comment:
As @bmc-msft suggested, it'd be nice to preserve consistent section ordering (when many are present). This is just alphabetical: "Added", "Changed", "Fixed".